.travis.yml

```diff
@@ -33,11 +33,11 @@ install:
   - pip install codecov
   - pip install coverage
   - pip install -r requirements.txt
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7"]]; then pip install -e azure-storage-nspkg; fi;  # Would work on Python 3, but will not be installed by PyPI. Trying to be close to real life.
```

Comment:
Apparently there's a syntax error:

/home/travis/.travis/job_stages: eval: line 98: syntax error in conditional expression: unexpected token `;'
/home/travis/.travis/job_stages: eval: line 98: syntax error near `;'
/home/travis/.travis/job_stages: eval: line 98: `if [[ "$TRAVIS_PYTHON_VERSION" == "2.7"]]; then pip install -e azure-storage-nspkg; fi; '